### ISSUE TITLE
FileNotFoundError when setting session through env var

### ISSUE THREAD

**Issue by FichteFoll** [wimglenn/advent-of-code-data#62]
When setting the session id through an env var (`export AOC_SESSION=…`), running `aocd` yielded the following exception:
```
$ aocd
Traceback (most recent call last):
  File "/home/mhm/code/advent-of-code/.venv/bin/aocd", line 8, in <module>
    sys.exit(main())
  File "/home/mhm/code/advent-of-code/.venv/lib/python3.9/site-packages/aocd/cli.py", line 50, in main
    data = get_data(day=args.day, year=args.year)
  File "/home/mhm/code/advent-of-code/.venv/lib/python3.9/site-packages/aocd/get.py", line 40, in get_data
    puzzle = Puzzle(year=year, day=day, user=user)
  File "/home/mhm/code/advent-of-code/.venv/lib/python3.9/site-packages/aocd/models.py", line 162, in __init__
    prefix = os.path.join(self.user.memo_dir, fname)
  File "/home/mhm/code/advent-of-code/.venv/lib/python3.9/site-packages/aocd/models.py", line 85, in memo_dir
    return os.path.join(AOCD_DATA_DIR, self.id)
  File "/home/mhm/code/advent-of-code/.venv/lib/python3.9/site-packages/aocd/models.py", line 72, in id
    with open(fname, "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/mhm/.config/aocd/token2id.json'
```
It worked when I dumped the session id into `~/.config/aocd/token`.

**Comment by kageru** [comment]
This also happened to me until I manually created `~/.config/aocd`, so I assume the current code doesn’t make sure that directory exists before writing to it.

**Comment by wimglenn** [comment]
Fixed in v1.1.1, thanks for the report!
